third_party_license_file_generator/__main__.py

```diff
@@ -131,7 +131,7 @@
     license_overrides = {}
     if args.license_override_file is not None:
         with codecs.open(args.license_override_file, "r", "utf-8") as f:
-            license_overrides = yaml.load(f.read())
+            license_overrides = yaml.load(f.read(), Loader=yaml.SafeLoader)
```

Comment:
👍🏼

Reply:
awesome - thank you very much for this helpful package! 😊